static/specification.html

```diff
@@ -138,605 +337,612 @@ <h2>
-              MonetizationProgressEventInit }} for more information.
-            </p>
-          </section>
-          <section>
-            <h2>
-              <dfn>receipt</dfn> attribute
-            </h2>
-            <p>
-              When getting, returns the value it was initialized with. See {{
-              MonetizationProgressEventInit / receipt }} member of {{
-              MonetizationProgressEventInit }} for more information.
-            </p>
-          </section>
-          <section data-dfn-for="MonetizationProgressEventInit">
-            <h3>
-              <dfn>MonetizationProgressEventInit</dfn> dictionary
-            </h3>
-            <pre class="idl">
-              dictionary MonetizationProgressEventInit : EventInit {
-                required DOMString amount;
-                required DOMString assetCode;
-                required DOMString assetScale;
-                DOMString receipt;
-              };
-            </pre>
-            <dl>
-              <dt>
-                <dfn>amount</dfn> member
-              </dt>
-              <dd>
-                The destination amount <em>received</em> as specified in the
-                ILP packet.
-              </dd>
-              <dt>
-                <dfn>assetCode</dfn> member
-              </dt>
-              <dd>
-                The three letter asset code describing the amount's units.
-              </dd>
-              <dt>
-                <dfn>assetScale</dfn> member
-              </dt>
-              <dd>
-                The scale of the amount. For example, USD would have an
-                <code>assetScale</code> of <code>2</code> when denominated in
-                cents.
-              </dd>
-              <dt>
-                <dfn>receipt</dfn> member
-              </dt>
-              <dd>
-                base64-encoded [[[Receipt]]] issued by the Web Monetization
-                receiver to the Web Monetization provider as proof of the total
-                amount received in the stream.
-              </dd>
-            </dl>
-          </section>
-        </section>
+        <p>
+          `null` or a base64-encoded [[[Receipt]]] issued by the [=monetization
+          receiver=] to the [=monetization provider=] as proof of the total
+          amount received in the [=payment session=]. When getting, returns the
+          value it was initialized with.
+        </p>
       </section>
-      <section id="http-headers">
+      <section>
         <h2>
-          HTTP Headers
+          <dfn>paymentPointer</dfn> attribute
         </h2>
-        <h4>
-          `Web-Monetization-Id`
-        </h4>
-        <p class="warning">
-          DEPRECATED: The `Web-Monetization-Id` header is deprecated in favor
-          of receipts.
+        <p>
+          A [=URL=] representing the [=payment pointer=] that has been
+          monetized. When getting, returns the value it was initialized with.
```

Comment:
I don't get what this is saying

Reply:
it's a getter, and returns the value passed in, in the constructor 
overly wordy imo